src/TicketBai/Mews.Fiscalizations.TicketBai/ModelToDtoConverter.cs

```diff
@@ -0,0 +1,395 @@
+            };
+        }
+
+        private static Sujetos Convert(Subject subject)
+        {
+            return new Sujetos
+            {
+                Emisor = Convert(subject.Issuer),
+                Destinatarios = subject.Receivers.Select(r => Convert(r)).ToArray(),
+                VariosDestinatariosSpecified = true,
+                VariosDestinatarios = subject.MultipleReceivers.Match(
+                    t => SiNoType.S,
+                    f => SiNoType.N
+                ),
+                EmitidaPorTercerosODestinatario = subject.IssuerType.Map(t => Convert(t)).ToNullable(),
+                EmitidaPorTercerosODestinatarioSpecified = subject.IssuerType.NonEmpty
+            };
+        }
+
+        private static Emisor1 Convert(Issuer issuer)
+        {
+            return new Emisor1
+            {
+                NIF = issuer.Nif.TaxpayerNumber,
+                ApellidosNombreRazonSocial = issuer.Name.Value
+            };
+        }
+
+        private static EmitidaPorTercerosType Convert(IssuerType type)
+        {
+            return type.Match(
+                IssuerType.IssuedByIssuer, _ => EmitidaPorTercerosType.N,
+                IssuerType.IssuedByThirdParty, _ => EmitidaPorTercerosType.T,
+                IssuerType.IssuedByTransactionRecipient, _ => EmitidaPorTercerosType.D
+            );
+        }
+
+        private static IDDestinatario Convert(Receiver receiver)
+        {
+            var name = receiver.Match(
+                local => local.Name,
+                foreign => foreign.Name
+            );
+            var address = receiver.Match(
+                local => local.Address,
+                foreign => foreign.Address
+            );
+            var postalCode = receiver.Match(
+                local => local.PostalCode,
+                foreign => foreign.PostalCode
+            );
+            return new IDDestinatario
+            {
+                ApellidosNombreRazonSocial = name.Value,
+                CodigoPostal = postalCode.Value,
+                Direccion = address.Value,
+                Item = receiver.Match(
+                    local => (object)local.TaxpayerIdentificationNumber.TaxpayerNumber,
+                    foreign => new IDOtro1
+                    {
+                        CodigoPais = foreign.Country.Map(c => CovertCountryType21(c)).ToNullable(),
+                        CodigoPaisSpecified = foreign.Country.NonEmpty,
+                        ID = foreign.Id.Value,
+                        IDType = Convert1(foreign.IdType)
+                    }
+                )
+            };
+        }
+
+        private static HuellaTBAI1 Convert(InvoiceFooter footer)
+        {
+            return new HuellaTBAI1
+            {
+                EncadenamientoFacturaAnterior = footer.OriginalInvoiceInfo.Map(i => Convert(i)).GetOrNull(),
+                Software = Convert(footer.Software),
```

Comment:
On the other hand, Software is something I wouldn't expect to be sending per invoice. But rather juts once per request.